Appraisals

```diff
@@ -0,0 +1,15 @@
+appraise "rails-4-0" do
```

Comment:
Happy to do so. Would you like to keep the separate Gemfiles for the different versions of Rails without appraisal though? So you can be sure that future changes don't break compatibility with older versions of Rails without having to manually test each one?